test/e2e/data/kubetest/conformance.yaml

```diff
@@ -1,7 +1,7 @@
 ginkgo.focus: \[Conformance\]
 ginkgo.skip: \[Serial\]
 disable-log-dump: true
-ginkgo.show-node-events: true
+ginkgo.progress: true
```

Comment:
Let's add a comment that this is still used as long as we run kubetest on versions <= v1.26.

Reply:
added the comment. Please check